NewSpring/ApollosProject
# [DEV-3716] Add Selected State on Menu

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds a selected state to the prayer menu.

Initial Menu:
![Screenshot 2019-04-29 14 34 46](https://user-images.githubusercontent.com/3229463/56918918-46119080-6a8d-11e9-81a8-91540d44ff60.png)

Selected Item:
![Screenshot 2019-04-29 14 34 51](https://user-images.githubusercontent.com/3229463/56918925-49a51780-6a8d-11e9-8a10-95ce030dda0d.png)

### How do I test this PR?
1. Run the app in your sim of choice and go to the prayer tab.
2. See the menu options.
3. Choose one. Your choice should be larger and have a darker color than the other choices.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3716](https://newspring.atlassian.net/browse/DEV-3716)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
